src/uk/gov/defra/ffc/Docker.groovy

```diff
@@ -1,10 +1,10 @@
 package uk.gov.defra.ffc
 
 class Docker implements Serializable {
-  static def buildTestImage(ctx, credentialsId, registry, projectName, buildNumber, tag) {
-    ctx.docker.withRegistry("https://$registry", credentialsId) {
-      ctx.sh("docker-compose -p $projectName-$tag-$buildNumber -f docker-compose.yaml -f docker-compose.test.yaml build")
-    }
+  static def buildTestImage(ctx, credentialsId, registry, projectName, buildNumber, tag) {    
```

Comment:
Some rogue whitespace added.  I think the indentation is slightly off by one space too.